server/controllers/AuthController.js

```diff
@@ -112,4 +111,56 @@ export default class AuthController {
       return responseMessage(response, 500, { message: error.message });
     }
   }
+
+/**
+ *
+ *
+ * @static
+ * @param {*} request
+ * @param {*} response
+ * @returns {object} - response object
+ * @memberof AuthController
+ */
+  static async verifyUserbyEmail(request, response) {
+    const { email } = request.body;
+    try {
+      const user = await User.findOne({ where: { email } });
+      if (!user) return responseMessage(response, 404, { status: 'failure', message: 'user does not exist' });
+      if (user) {
```

Comment:
you dont need if statement here because it will only come here if there is a user

Reply:
noted